packages/frameworks/src/frameworks.ts

```diff
@@ -202,12 +202,13 @@ export const frameworks = [
     useRuntime: { src: 'package.json', use: '@vercel/remix' },
     ignoreRuntimes: ['@vercel/node'],
     detectors: {
-      every: [
-        // Intentionally does not detect a package name
-        // https://github.com/vercel/vercel/pull/7761
+      some: [
         {
           path: 'remix.config.js',
         },
+        {
+          path: 'remix.config.mjs',
+        },
```

Comment:
Let's separate this change into a different PR because the changes won't take affect until its updated in `vercel/api`

Reply:
Sounds good.

Reply:
https://github.com/vercel/vercel/pull/9558